website/docs/cdktf/community.mdx

```diff
@@ -1,11 +1,11 @@
 ---
-page_title: CDKTF Community - CDK for Terraform
+page_title: The CDK for Terraform community
```

Comment:
I think we should use "CDK community" instead of "Terraform community" because we aren't talking about the "Terraform community edition" here:

```suggestion
page_title: The CDK community for Terraform
```